spec/how_is_spec.rb

```diff
@@ -49,7 +49,7 @@
 
           VCR.use_cassette("how-is-with-config-file") do
             expect {
-              reports = HowIs.from_config(YAML.load_file(HOW_IS_CONFIG_FILE))
+              reports = HowIs.from_config(YAML.load_file(HOW_IS_CONFIG_FILE), "2017-08-01")
```

Comment:
Note to self: Consider making `HowIs.from_config()` use a `start_date` keyword argument like `HowIs.new()` does, for consistency.